queries/slint/highlights.scm

```diff
@@ -11,7 +11,7 @@
 (function_identifier) @function
 
 (reference_identifier) @keyword
-(visibility_modifier) @keyword
+(visibility_modifier) @include
```

Comment:
I assume that `visibility_modifier` is something like public/private. This is usually not used in such a way in other languages. Or is the visibility_modifier directly used on import?

Reply:
Gotcha. For now it just references `export` but it's used in a similar way to public/private. In that case I'll change it back

Reply:
Ok, then you were probably right. `export` should be `@include`. Sorry for the confusion.